_posts/2020-03-09-our-core-values.markdown

```diff
@@ -0,0 +1,123 @@
+to live" a great life, and to grow as professionals within the company.
+
+In terms of our communication, we adhere to the contributor covenant v2.0, not 
+just for our open source projects, but also for any interaction we have with 
+team members: [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md)
+
+## Challenging Projects over Profitable Projects
+
+We value interesting challenges more than high-paying contracts. We are in the 
+problem-solving business for real companies. We won't work on _cool-sounding 
+solutions_ looking for a problem that doesn't exist. We will only work with 
+businesses and entrepreneurs who have real problems. The only way to be great 
+problem solvers is to understand and believe in our clients' businesses.
+
+## Talent is Everywhere
+
+The value of our team will always be higher than the sum of the value of each 
+team member. There is value in team synergy and our interactions. In sports, 
+team work beats raw talent every time. This applies equally in our work 
+environment.
+
+We won't let geographical boundaries stop us from hiring the best person for 
+the job. As long as our team members can communicate properly, distance won't 
+be an issue.
+
+There is great value in diversity and we must strive to foster a diverse 
+workforce.
+
+## Open by Default
+
+Everything we do should be open by default. We want to have open communications, 
+contribute to open source projects, give back to our communities, and become 
+thought leaders in our industry. Openness is key to our leadership. When starting
+a project, we should ask ourselves: "Is there any reason to make this a closed
+source project?"
+
+There must be a solid reason to start a new closed source project. Otherwise, 
+our projects should be open by default.
+
+## Continuous Improvement
+
+We are here to pay off technical debt and invest in our clients' assets. We will 
+always strive to leave legacy systems in better shape than when we started 
+working on them.
+
+This does not simply apply to code. It also applies to our clients' culture,
+processes, dependencies, code coverage, documentation, best practices, and 
+anything that will make our client's life easier in the long term.
+
+We are here to learn from our mistakes. The only unforgivable mistake is to 
+not learn from our mistakes.
+
+# Other Values
+
+We find value in other ideas, practices, and techniques. To name a few:
+
+## Agile Methodologies 
+
+We want to apply agile methodologies in every project we work on. Some projects 
+might require us to implement SCRUM, others might be better suited for XP. As an
+agile team we must always keep this in mind:
+
+> Individuals and interactions over processes and tools
+
+## Pair Programming 
+
+We don't commit to pair program 100% of the time. We commit to do pair 
+programming when it's the best course of action. We find a lot of value in this 
+technique as a way to share knowledge, find the best solution, and produce 
+high-quality code.
```

Comment:
I think it might be good to emphasize that it helps us, as a remote-friendly / remote-first company, to humanize our co-workers a bit as well.  When you pair and work on a problem, the person you're working with becomes less of a remote endpoint and more a human being.

Reply:
Just addressed this. 👍